src/pages/@owner/food-truck-management/FoodTruckManagement.tsx

```
@@ -0,0 +1,118 @@
+        className={cn(
+          data.length > 0 &&
+            'fixed bottom-[0] mx-auto w-full max-w-[60rem] bg-white px-[2rem] py-[1.7rem]',
+          'px-[2rem]',
+          isScrolled && 'shadow-[0_-4px_10px_0_rgba(0,0,0,0.04)]'
+        )}
+      >
+        <Button
+          variant='default'
+          buttonStyle='large'
+          children='+ 추가하기'
+          handleClickButton={handleNavigateToAdd}
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)